### PR TITLE
Fill zero in front of signature array element before encode

### PR DESCRIPTION
**Pull request by jimni1222**
## Proposed changes

Fill zero in front of signature array element before encode.

## Types of changes

Please put an x in the boxes related to your change.

- [x] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-js/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-js)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

close https://github.com/klaytn/caver-js/issues/12

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: test/packages/caver.klay.accounts.js [hunk 1]
```diff
@@ -311,6 +311,30 @@ describe('caver.klay.accounts.recoverTransaction', () => {
       expect(result).to.not.equal(account.addrss)
     })
   })
+
+  context('CAVERJS-UNIT-WALLET-104 : rawTransaction:made by signature trimed leading zero', () => {
```
Comment (kjhman21) [inline test/packages/caver.klay.accounts.js]:
```suggestion
  context('CAVERJS-UNIT-WALLET-104 : rawTransaction: leading zeroes of the signature are trimmed', () => {
```

File: packages/caver-klay/caver-klay-accounts/src/index.js [hunk 2]
```diff
@@ -315,7 +315,16 @@ Accounts.prototype.signTransactionWithSignature = function signTransactionWithSi
  */
 Accounts.prototype.recoverTransaction = function recoverTransaction(rawTx) {
     var values = RLP.decode(rawTx);
-    var signature = Account.encodeSignature(values.slice(6,9));
+
+    // If the leading zero is trimmed, it will be filled with a valid length of '0'.
+    const arr = values.slice(7,9).map((sig) => {
```
Comment (kjhman21) [inline packages/caver-klay/caver-klay-accounts/src/index.js]: This slicing works only if the raw Tx is a legacy transaction. Is this okay?

Comment (jimni1222) [reply packages/caver-klay/caver-klay-accounts/src/index.js]: oh, i did't think about other transaction type.
original recoverTransaction logic cannot handle other transaction type ;-;... 
This has to be implemented with various transaction type... ! 
I will implement that !! Thank you for reviewing ;-;..

Comment (jimni1222) [reply packages/caver-klay/caver-klay-accounts/src/index.js]: @kjhman21 Error and test code is added :) Please take a look !

**Pull request merged**